t/plugin/request-id.t

```diff
@@ -387,3 +387,86 @@ GET /t
 request header not present
 --- no_error_log
 [error]
+
+
+
+=== TEST 10: add plugin with custom header name in global rule and add plugin with default header name in specific route
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/global_rules/1',
+                ngx.HTTP_PUT,
+                     [[{
+                        "plugins": {
+                            "request-id": {
+                                "header_name":"Custom-Header-Name"
+                            }
+                        }
+                }]]
+                )
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                    [[{
+                        "plugins": {
+                            "request-id": {
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1982": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/opentracing"
+                }]]
+                )
+            if code >= 300 then
+                ngx.status = code
+                return
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
```

Comment:
Please fix the linter: https://github.com/apache/apisix/pull/4479/checks?check_run_id=2929454713

Reply:
there should be 3 blank lines between test cases, or you can use the reindex to do. the CI error is about this.